docs/asus-aura-led-guide.md

```diff
@@ -0,0 +1,92 @@
+
+__NOTE:__
+Some features of this driver are still being worked on. Your use of this driver indicates that you understand and accept it is a beta release.
+
+
+This driver supports Asus Aura USB-based lighting controllers that appear in Asus Z690 motherboards. These controllers operate in either (a) direct mode or (b) effect mode. _Direct_ mode is employed by Aura Crate in Windows. It requires the application to send a continuous stream of commands to the controller in order to modulate the lighting on each addressable LED in each channel. The other mode is _effect_ mode in which the controller itself modulates the lighting on each addressable LED in each channel. Effect mode requires the application to send a single set of command codes to the controller in order to initiate the given effect. The controller handles the rest until such time that the application sends a different command set.
+
+This driver employs the _effect_ mode (fire and forget). The selected lighting mode remains in effect until it is explicitly changed. This means the selected lighting mode remains in effect (a) on cold boot, (b) on warm boot, (c) after wake-from-sleep.
+
+The disadvantage, however, is the inability to set different lighting modes to different lighting channels. All channels remain synchronized.
+
+There are three known variants of the Aura LED USB-based controller:
+
+- Device `0x19AF`: found in Asus ProArt Z690-Creator WiFi
+- Device `0x1939` [^1]
+- Device `0x18F3`[^1]: found in Asus ROG Maximus Z690 Formula
+
+[^1]: Support for devices `0x1939` and `0x18F3` is not properly or sufficiently developed so these devices have been commented-out in the driver. Users may uncomment the line(s) in `SUPPORTED_DEVICES` to experiment and provide feedback. Wireshark USB traffic  capture, in particular, will be very helpful.
+
+
+## Initialization
+
+Asus Aura LED controller does not need to be initialized before use. Initialization is optional and recommended.
+
+```
+# liquidctl -m Aura initialize
+AsusTek Aura LED Controller
+└── Firmware version    AULA3-AR32-0207
+```
+
+## Status
+
+The `status` function currently returns a list of 6-byte values whose meanings are not fully understood. This information is provided nevertheless in the hopes of encouraging someone to interpret them properly.
+```
+% liquidctl -m Aura status
+AsusTek Aura LED Controller
+├── Device Config: 1     0x1e, 0x9f, 0x03, 0x01, 0x00, 0x00
+├── Device Config: 2     0x78, 0x3c, 0x00, 0x01, 0x00, 0x00
+├── Device Config: 3     0x78, 0x3c, 0x00, 0x01, 0x00, 0x00
+├── Device Config: 4     0x78, 0x3c, 0x00, 0x00, 0x00, 0x00
+├── Device Config: 5     0x00, 0x00, 0x00, 0x01, 0x04, 0x02
+├── Device Config: 6     0x01, 0xf4, 0x00, 0x00, 0x00, 0x00
+├── Device Config: 7     0x00, 0x00, 0x00, 0x00, 0x00, 0x00
+├── Device Config: 8     0x00, 0x00, 0x00, 0x00, 0x00, 0x00
+├── Device Config: 9     0x00, 0x00, 0x00, 0x00, 0x00, 0x00
+└── Device Config: 10    0x00, 0x00, 0x00, 0x00, 0x00, 0x00
```

Comment:
Make it so that this output is only shown with the `--debug` flag since it is currently here for development / debugging purposes.

Reply:
Also make sure to comment in the documentation that this is unstable.